services/agents/registry.go

```diff
@@ -127,6 +129,10 @@ func (r *Registry) Run(stream agentpb.Agent_ConnectServer) error {
 		l.Infof("Disconnecting client: %s.", disconnectReason)
 	}()
 
+	if err := r.sendAgentMetadata(stream, agent.id); err != nil {
```

Comment:
maybe it's better to send the whole agent or just agent.RunsOnNodeID instead of agentID to not fetch it from DB again?

Reply:
oh, wait